compose/django.md

```diff
@@ -60,23 +60,31 @@ and a `docker-compose.yml` file. (You can use either a `.yml` or `.yaml` extensi
     expose. See the [`docker-compose.yml` reference](/compose/compose-file/index.md) for more
     information on how this file works.
 
-9.  Add the following configuration to the file.
+9. Add the following configuration to the file.
 
     ```none
-    version: '3'
-
-    services:
-      db:
-        image: postgres
-      web:
-        build: .
-        command: python manage.py runserver 0.0.0.0:8000
-        volumes:
-          - .:/code
-        ports:
-          - "8000:8000"
-        depends_on:
-          - db
+    # docker-compose.yml
```

Comment:
Please, remove this comment line. The previous steps already mention the filename.

Reply:
Okay